.github/workflows/build-test-release.yml

```diff
@@ -40,7 +40,10 @@ jobs:
           CGO_CFLAGS: "-O -D__BLST_PORTABLE__" # Set the CGO flags to use the portable version of BLST
   e2e_test:
     name: e2e tests
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+            os: [ubuntu-20.04, macos-12]
```

Comment:
What is the reason you suggest to set to explicit 20.04 vs the current `ubuntu-latest`?

Reply:
ok let's use 20.04 and macos-12, to provide an environment most compatible with avalanchego binaries